vitrina/api/urls.py

```diff
@@ -13,6 +15,68 @@
     path('partner/api/1/', get_partner_schema_view().with_ui('redoc'), name='partner-api'),
     path('partner/api/1/', include(router.urls)),
 
+    # dataset api urls
+    path('partner/api/1/datasets/', DatasetViewSet.as_view({
```

Comment:
```suggestion
    path('partner/api/1/datasets', DatasetViewSet.as_view({
```

Original API did not have trailing slashes, so we should be fully backwards compatible. Same thing applies to other routes.